modules/core/test/spec/cuba/core/data_manager/DataManagerCommitTest.groovy

```diff
@@ -36,91 +36,120 @@ class DataManagerCommitTest extends Specification {
 
     private DataManager dataManager
     private EntityStates entityStates
+    private Customer customer
 
     void setup() {
+        initBeans()
+
+        customer = new Customer(name: 'Smith')
+    }
+
+    protected void initBeans() {
         dataManager = AppBeans.get(DataManager)
         entityStates = AppBeans.get(EntityStates)
     }
 
-    def "usage of returned entities"() {
+    def "commited, returned entities allow to get entity by its ID"() {
```

Comment:
Plz fix the method names.
"committed"
"allow _you_ to" (never just "allow to")

Reply:
done